float64 altitude
float64 pressure
float64 qnh
uint64 timeStamp